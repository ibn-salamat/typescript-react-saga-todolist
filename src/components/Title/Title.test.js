import { shallow } from "enzyme";
import { Title } from ".";

const setup = (props) => shallow(<Title {...props} />);


describe("Title", () => {
  it("should render component", () => {
    const component = setup();
    expect(component).toMatchSnapshot();
  });

  it("it should contain label", () => {
    const label = "My Title";
    const component = setup({ label });

    expect(component.find("p").text()).toBe(label);
  });
});
