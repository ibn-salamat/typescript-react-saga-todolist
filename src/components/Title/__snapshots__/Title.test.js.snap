// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render component 1`] = `
<p
  style={
    Object {
      "fontSize": 24,
    }
  }
/>
`;
